crates/sncast/src/starknet_commands/declare.rs

```diff
@@ -49,8 +49,11 @@ pub struct Declare {
     pub rpc: RpcArgs,
 }
 
+#[expect(clippy::too_many_arguments)]
 pub async fn declare(
-    declare: Declare,
+    contract: String,
```

Comment:
nit: In consistency with flag name:
```suggestion
    contract_name: String,
```

Reply:
Why was it resolved?